### PR TITLE
use shields.io image for zenhub badge; closes #8

### LINKED ISSUE #8: zenhub svg broken in readme on github.com
It looks like github isn't serving a good mime type for svgs coming out of other repos. Maybe we should use the shields.io version after all. ¯\\\_(ツ)_/¯

### PR DESCRIPTION
**Pull request by infinitewarp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 [![license](https://img.shields.io/github/license/cloudigrade/cloudigrade.svg)]()
 [![Build Status](https://travis-ci.org/cloudigrade/cloudigrade.svg?branch=master)](https://travis-ci.org/cloudigrade/cloudigrade)
 [![Coverage Status](https://coveralls.io/repos/github/cloudigrade/cloudigrade/badge.svg?branch=coverage-configs)](https://coveralls.io/github/cloudigrade/cloudigrade?branch=coverage-configs)
-[![Scrum Board](https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.svg)](https://app.zenhub.com/workspace/o/cloudigrade/cloudigrade/boards?repos=118967787)
+[![Scrum Board](https://img.shields.io/badge/Shipping_faster_with-ZenHub-5e60ba.svg?style=flat)](https://app.zenhub.com/workspace/o/cloudigrade/cloudigrade/boards?repos=118967787)
```
Comment (ilyavw) [inline README.md]: Test

File: README.md [hunk 2]
```diff
@@ -3,4 +3,4 @@
 [![license](https://img.shields.io/github/license/cloudigrade/cloudigrade.svg)]()
 [![Build Status](https://travis-ci.org/cloudigrade/cloudigrade.svg?branch=master)](https://travis-ci.org/cloudigrade/cloudigrade)
 [![Coverage Status](https://coveralls.io/repos/github/cloudigrade/cloudigrade/badge.svg?branch=coverage-configs)](https://coveralls.io/github/cloudigrade/cloudigrade?branch=coverage-configs)
-[![Scrum Board](https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.svg)](https://app.zenhub.com/workspace/o/cloudigrade/cloudigrade/boards?repos=118967787)
```
Comment (ilyavw) [inline README.md]: Test 2

**Comment by ilyavw** [comment]
Test

File: README.md (hunk 1 above)
Comment (ilyavw) [inline README.md]: Can we have a spicier tagline than this?

Comment (infinitewarp) [reply README.md]: If we're going to link to them, I figured we should keep their official tagline that's in their badge.

**Pull request merged**